images/win/scripts/Installers/Install-Sbt.ps1

```diff
@@ -2,8 +2,6 @@
 ##  File:  Install-Sbt.ps1
 ##  Desc:  Install sbt for Windows
 ################################################################################
-$ErrorActionPreference = "Stop"
-
 Import-Module -Name ImageHelpers
```

Comment:
May be we should also remove `Import-Module -Name ImageHelpers`?
It doesn't make much sense since all functions are auto-imported

Reply:
Done.